content/general/concepts/software-engineering/software-engineering.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Software Engineering'
+Description: 'Software engineering is a field of study that involves using engineering principles to design, build, test, and maintain software systems.'
+Subjects:
+  - 'Mobile Development'
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Development'
+  - 'Servers'
+  - 'Frameworks'
+  - 'Developer Tools'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+  - 'paths/back-end-engineer-career-path'
```

Comment:
```suggestion
```

Reply:
We limit the subject and content fields to two items, tags can contain as many as five